tests/backend/mongodb/test_admin.py

```diff
@@ -30,14 +30,22 @@ def mock_replicaset_config():
     }
 
 
-def test_add_replicas(mock_replicaset_config):
+@pytest.fixture
+def connection():
```

Comment:
I would just rename this to something like `connected_connection` to be painfully obvious that it's connected ;)